### DISCUSSION TITLE
Welcome to YOMEN Discussions!

### DISCUSSION THREAD

**Discussion by fdciabdul** [tegal1337/YOMEN#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yorukhun** [comment]
How to using only for live streams ?

**Reply by fdciabdul** [reply]
it's pretty simple , maybe i'll add this one in the next update
